src/Persisters/Traits/ResolveValuesHelper.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ORM\Persisters\Traits;
+
+use BackedEnum;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Proxy\DefaultProxyClassNameResolver;
+
+use function array_merge;
+use function is_array;
+use function is_object;
+
+trait ResolveValuesHelper
+{
+    protected EntityManagerInterface $em;
```

Comment:
We don't need to duplicate this property. Please remove it.

Reply:
Removed